tests/test_grid.py

```diff
@@ -11,8 +11,8 @@
 dir_path = os.path.join(data_dir, 'dir.asc')
 dem_path = os.path.join(data_dir, 'dem.tif')
 roi_path = os.path.join(data_dir, 'roi.tif')
-eff_path = os.path.join(data_dir, 'eff.tif')
-dinf_eff_path = os.path.join(data_dir, 'dinf_eff.tif')
+# eff_path = os.path.join(data_dir, 'eff.tif')
```

Comment:
Can you delete these commented-out lines? The old lines will still be in the history if needed.